lesson_01.md

```diff
@@ -24,7 +24,7 @@ Students will be able to...
 | 10 minutes | Course staff introductions                  |
 | 10 minutes | Icebreakers                                 |
 | 10 minutes | Course outline                              |
-| 15 minutes | Course norms, procedures, and administrivia |
+| 15 minutes | Course norms, procedures, and administrative|
```

Comment:
I believe the original word is intended as-is — it's the colloquialism for _administrative trivia_. There's room, however, for discussion about whether the curriculum should avoid such forms, particularly for readability across an audience with that includes non-native English speakers.

Reply:
The original word is intentional.  However it wording could diminish the importance of doing the administrative part of the start of class procedures.  Since the TEALS curriculum is intended for a wide audience, I'm inclined to accept this pull request.  I'm open to other's point of view.

Reply:
Note that the correction is grammatically incorrect. It could be "administrative tasks", or "administration". It needs to be a noun.